src/solver/multiphysics_integrator.h

```diff
@@ -118,6 +120,21 @@ namespace solver
         auto nbrOfLevels() const { return nbrOfLevels_; }
 
 
+        void registerTagger(int coarsestLevel, int finestLevel,
+                            std::unique_ptr<PHARE::amr::Tagger> tagger)
```

Comment:
does "std::unique_ptr<PHARE::amr::Tagger> && tagger" work better?

the notion of a copy of a unique_ptr doesn't make much sense

Reply:
if we every tried to copy a unique_ptr it would error at compile so maybe it doesn't matter